content/en/blog/_posts/2022-01-03-kubernetes-is-moving-on-from-dockershim.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: blog
+title: "Kubernetes is moving on from Dockershim: commitments and next steps"
+date: 2022-01-03
+slug: kubernetes-is-moving-on-from-dockershim
+---
+
+**Authors:** Sergey Kanzhelev (Google), Jim Angel (Google), Davanum Srinivas (VMware), Shannon Kularathna (Google), Chris Short (AWS), Dawn Chen (Google)
+
+Kubernetes is removing dockershim in the upcoming v1.24 release. We're excited
+to reaffirm our community values by supporting open source container runtimes,
+enablin a smaller kubelet, and faster engineering velocity for teams using
+Kubernetes. If you [use Docker Engine as a Container Runtime](/docs/tasks/administer-cluster/migrating-from-dockershim/find-out-runtime-you-use/)
+for your Kubernetes cluster, get ready to migrate to 1.24! To check if you're
+affected, refer to [Check whether dockershim deprecation affects you](/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you/).
+
+## Why we’re moving away from dockershim
+
+Docker was the first container runtime used by Kubernetes. This is one of the
+reasons why Docker is so familiar to many Kubernetes users and enthusiasts.
+Later, we wanted to support other container runtimes. Today, Kubernetes uses the
+container runtime interface (CRI) to let system components (like the kubelet)
+talk to container runtimes. That original, baked in support for Docker turned
+into the component we call dockershim. It was built into Kubernetes code.
+Removing dockershim from Kubernetes inevitably affects the entire CNCF family of projects beyond
+Kubernetes. Dependencies on Docker and dockershim have crept into various tools
+and projects in the Kubernetes ecosystem.
+
+There are a few container runtimes today that are often lighter, faster, and
+more secure than when Docker Engine was the only choice. By removing the
+dockershim CRI, we're embracing the first value of CNCF: "[Fast is better than
+slow](https://github.com/cncf/foundation/blob/master/charter.md#3-values)".
+There are a number of technical, historical, and community-oriented reasons why
+we think this is the best move for the Kubernetes project that we'll explore in
+detail in a future post. Stay tuned for future communications on the topic!
```

Comment:
```suggestion
Stay tuned for future communications on the topic!
```

Your conclusion with the first value of CNCF is _way_ stronger by removing this sentence!